### DISCUSSION TITLE
关于 logo 设计的讨论

### DISCUSSION THREAD

**Discussion by DuanPengfei** [artusjs/spec#19] [Ideas]
- logo 的风格？像素风？
- logo 的含义？art 标识艺术，引申出优雅？us 标识我们，引申出人，开放，合作？

**Comment by atian25** [comment]
像素风 = art，直接像素风的字体 + 颜色就可以了。

**Reply by DuanPengfei** [reply]
![image](https://user-images.githubusercontent.com/4469502/167335761-26a6cbf9-b6f2-4bb5-9f68-94e8e2dc2112.png)
类似于这种的感觉？（可能细致点每个字本身是通过像素点，每个像素点可以单独设置颜色，我这里直接用的字体，demo 还做不到）

**Reply by DuanPengfei** [reply]
顺着这个风格，是不是可以用乐高设计器做一下类似乐高风格的？像素风有了，也有了框架、组合的含义。

**Comment by XadillaX** [comment]
Artus = 骨架、架构、四肢。抽象一个人的四肢，用几何表达。

**Comment by thonatos** [comment]
风格、思路都有了，哪里可以“抓”个设计师搞起，😁
